### PR TITLE
Updated new file

### PR DESCRIPTION
**Pull request by Pratap304**
Please review the code

### CODE REVIEW

File: file [hunk 1]
```diff
@@ -1,2 +1,3 @@
 What is the currency in US?
 USD
+And What is currency of Germany?
```
Comment (anudeepwork) [inline file]: please add the name of currency

**Review by anudeepwork — CHANGES_REQUESTED** [review]
please make the changes

**Comment by Pratap304** [comment]
Please review the updated

File: file [hunk 2]
```diff
@@ -1,3 +1,4 @@
 What is the currency in US?
 USD
-And What is currency of Germany?
\ No newline at end of file
+And What is currency of Germany?
+euro
```
Comment (anudeepwork) [inline file:4]: good to merge

**Review by anudeepwork — COMMENTED** [review]
Good to merge the changes

**Pull request merged**